### DISCUSSION TITLE
Segmentation fault when reading parameter file

### DISCUSSION THREAD

**Discussion by argirios44** [SeisSol/SeisSol#682] [Q&A] [answered]
Hello, i have been trying to compile SeisSol and run the first example tpv33 for a while now with no result. I have tried multiple HOST_ARCH and Orders but always results in errors when trying to run them, segmentation faults for the most part. Seems i am close but i still cannot get it to run properly. Any help is appreciated. :-)

Update: After running in debug mode it seems the error happens when trying to read the parameter file.

![image](https://user-images.githubusercontent.com/35249593/193454994-b29a91f1-bbaf-4cb6-97a1-e01a7b702936.png)

[debugOutput.txt](https://github.com/SeisSol/SeisSol/files/9692546/debugOutput.txt)

CPU: AMD Ryzen™ 5 5600X
RAM: 32 GB

Thanks in advance

**Comment by sebwolf-de** [comment]
Hi, for your CPU probably the `hsw` option is best. Could you post your parameters file?

**Comment by argirios44** [comment]
Hi, i am using the parameter file from this example https://github.com/SeisSol/Examples/tree/master/tpv33 without editing it.

**Comment by sebwolf-de** [comment, marked as answer]
I tried the examples setup together with the latest master and it did work for me, when I fixed the mesh file name: https://github.com/SeisSol/Examples/pull/53.
The line `ParsingError: cannot find the end of block:         &copy; 2022 GitHub, Inc.` indicates that you have the string `2022 GitHub, Inc.` somewhere in your parameter file. Could that be?

**Comment by argirios44** [comment]
You are right, it turns out i was actually downloading the linked github page of the parameter file which had the whole github html wrapping the parameters...
Thanks a lot, it's reading the file now!
